CHANGELOG.rst

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 ==========
 
 * doc-fix: Remove incorrect parameter for EI TFS Python README
+* enhancement: tox.ini: Add twine check for PyPI README
```

Comment:
actually, upon seeing @nadiaya's comment on #653, I think it'd make sense to at least remove the "enhancement" label here.